src/style.css

```diff
@@ -0,0 +1,67 @@
+body {
+  padding: 0;
+  margin: 0;
+  background-color: #d3d3d3;
+}
+
+li {
+  list-style: none;
+  display: flex;
+  border-bottom: 0.01px solid #d3d3d3;
+  margin-left: -4%;
+}
+
+.item-list {
+  display: flex;
+  gap: 20px;
+  width: 100%;
+  justify-content: flex-start;
+}
+
+.toDoList {
+  margin: 10%;
+  width: 60%;
+  box-shadow: 0 4px 8px 0 rgba(0, 0, 0, 0.2);
+  padding: 10px 10px;
+  background-color: #fff;
+}
+
+.refresh {
+  border-bottom: 0.01px solid #d3d3d3;
+}
+
+.add-item {
+  border-bottom: 0.01px solid #d3d3d3;
+}
+
+.add-activity {
+  border: none;
+  font-size: 16px;
+  padding: 15px;
+  width: 94%;
+}
+
+ul {
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-start;
+  justify-items: flex-start;
+}
+
+.clear {
+  text-align: center;
+}
+
+.clearBtn {
+  border: none;
+  background-color: #fff;
+  cursor: pointer;
+  text-align: center;
+  width: 100%;
+  font-weight: 400;
+  padding: 15px;
+}
+
+.clearBtn:hover {
+  background-color: #d3d3d3;
+}
```

Comment:
- [ ] Please make sure that your design CSS is a bit similar to the design in the [captured video](https://www.youtube.com/watch?v=AcUd-_Yjjqg) and please center your todo list to give a good UI as the one in the video shared with you :smile: